src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -384,26 +384,21 @@ private void Browser_CoreWebView2InitializationCompleted(object sender, CoreWebV
 
             try
             {
-                 this.browser.NavigateToString(libraryHTMLPage);
+                this.browser.NavigateToString(libraryHTMLPage);
+                SetLibraryFontSize();
+                SetTooltipText();
+                browser.ZoomFactor = (double)dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale / 100d;
+                browser.ZoomFactorChanged += Browser_ZoomFactorChanged;
+                browser.KeyDown += Browser_KeyDown;
+
+                // Hosts an object that will expose the properties and methods to be called from the javascript side
+                browser.CoreWebView2.AddHostObjectToScript("scriptObject",
+                    new ScriptObject(OnCopyToClipboard, OnPasteFromClipboard));
             }
             catch (Exception ex)
             {
                 string msg = ex.Message;
```

Comment:
Should log the msg here `LogToDynamoConsole`

Reply:
Yup, I can add it

Reply:
@QilongTang I wonder if we need to do the same for NotificationCenter 
It subscribes to the same event as libraryView